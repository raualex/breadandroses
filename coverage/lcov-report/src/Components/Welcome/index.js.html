<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/Components/Welcome/index.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">src/Components/Welcome</a> index.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>2/2</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import './Welcome.css';
&nbsp;
export const Welcome = () =&gt; {
&nbsp;
  return (
    &lt;div className="welcome-container"&gt;
      &lt;h1 className="welcome-title"&gt;WELCOME TO &lt;span&gt;BREAD &amp; ROSES&lt;/span&gt;&lt;/h1&gt;
      &lt;p className="welcome-body"&gt;Your one-stop-shop for government accountability for Laborers and the American Working Class.&lt;/p&gt;
      &lt;p className="welcome-body"&gt;Navigate above to see who is in your Senate and House Labor Committees and what they are doing for you.&lt;/p&gt;
    &lt;/div&gt;
  )
}
&nbsp;
export default Welcome;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Mon Nov 05 2018 11:35:14 GMT-0700 (MST)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
