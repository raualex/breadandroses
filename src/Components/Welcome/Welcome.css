span {
  color: #CC2B1D;
}

.welcome-body {
  font-family: 'Black Han Sans', sans-serif;
  margin: 26px;
}

.welcome-container {
  background-color: rgba(243,243,243,0.8);
  border: solid 6px #000;
  border-radius: 48px;
  margin: 8px;
  padding: 20px;
}

.welcome-title {
  -webkit-text-stroke: 2px #000;
  color: #f4c076;
  font-family: 'Black Han Sans', sans-serif;
  font-size: 2.5em;
}
