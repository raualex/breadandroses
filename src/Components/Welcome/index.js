import React from 'react';
import './Welcome.css';

export const Welcome = () => {

  return (
    <div className="welcome-container">
      <h1 className="welcome-title">WELCOME TO <span>BREAD & ROSES</span></h1>
      <p className="welcome-body">Your one-stop-shop for government accountability for Laborers and the American Working Class.</p>
      <p className="welcome-body">Navigate above to see who is in your Senate and House Labor Committees and what they are doing for you.</p>
    </div>
  )
}

export default Welcome;