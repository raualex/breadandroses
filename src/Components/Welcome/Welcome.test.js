import React from 'react';
import Welcome from './';
import { shallow } from 'enzyme';

describe('Welcome', () => {

  it('should match the snapshot', () => {
    
    let wrapper = shallow(<Welcome />)

    expect(wrapper).toMatchSnapshot()
  });
});