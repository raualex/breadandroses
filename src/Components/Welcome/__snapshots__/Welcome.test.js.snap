// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Welcome />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="welcome-title"
        >
          WELCOME TO 
          <span>
            BREAD & ROSES
          </span>
        </h1>,
        <p
          className="welcome-body"
        >
          Your one-stop-shop for government accountability for Laborers and the American Working Class.
        </p>,
        <p
          className="welcome-body"
        >
          Navigate above to see who is in your Senate and House Labor Committees and what they are doing for you.
        </p>,
      ],
      "className": "welcome-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "WELCOME TO ",
            <span>
              BREAD & ROSES
            </span>,
          ],
          "className": "welcome-title",
        },
        "ref": null,
        "rendered": Array [
          "WELCOME TO ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "BREAD & ROSES",
            },
            "ref": null,
            "rendered": "BREAD & ROSES",
            "type": "span",
          },
        ],
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Your one-stop-shop for government accountability for Laborers and the American Working Class.",
          "className": "welcome-body",
        },
        "ref": null,
        "rendered": "Your one-stop-shop for government accountability for Laborers and the American Working Class.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Navigate above to see who is in your Senate and House Labor Committees and what they are doing for you.",
          "className": "welcome-body",
        },
        "ref": null,
        "rendered": "Navigate above to see who is in your Senate and House Labor Committees and what they are doing for you.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="welcome-title"
          >
            WELCOME TO 
            <span>
              BREAD & ROSES
            </span>
          </h1>,
          <p
            className="welcome-body"
          >
            Your one-stop-shop for government accountability for Laborers and the American Working Class.
          </p>,
          <p
            className="welcome-body"
          >
            Navigate above to see who is in your Senate and House Labor Committees and what they are doing for you.
          </p>,
        ],
        "className": "welcome-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "WELCOME TO ",
              <span>
                BREAD & ROSES
              </span>,
            ],
            "className": "welcome-title",
          },
          "ref": null,
          "rendered": Array [
            "WELCOME TO ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "BREAD & ROSES",
              },
              "ref": null,
              "rendered": "BREAD & ROSES",
              "type": "span",
            },
          ],
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Your one-stop-shop for government accountability for Laborers and the American Working Class.",
            "className": "welcome-body",
          },
          "ref": null,
          "rendered": "Your one-stop-shop for government accountability for Laborers and the American Working Class.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Navigate above to see who is in your Senate and House Labor Committees and what they are doing for you.",
            "className": "welcome-body",
          },
          "ref": null,
          "rendered": "Navigate above to see who is in your Senate and House Labor Committees and what they are doing for you.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
